test/test_autograd.py

```diff
@@ -2697,7 +2697,6 @@ def test_var_mean_differentiable(self):
         torch.autograd.backward(r2, grad)
         self.assertEqual(input1.grad, input2.grad, rtol=0.01, atol=0.0)
 
-    @slowTest
```

Comment:
Not slow anymore? This was forgotten in the gradcheck speed up?

Reply:
It takes 3 seconds in my environment, not 1-2 minutes from long ago.

Reply:
Awesome!